bot/discord/commands/votemute.js

```diff
@@ -0,0 +1,64 @@
+const ms = require('ms');
+const Discord = require('discord.js');
+const db = require('quick.db');
+const mutesData = new db.table("muteData");
+let mutes = {};
+
+exports.run = async(client, message, args) => {
+  let modlog = message.guild.channels.cache.find(channel => channel.id === config.DiscordBot.modLogs);
+  if (!message.guild.me.hasPermission('MANAGE_ROLES')) return message.reply('Sorry, i dont have the perms to do this cmd i need MANAGE_ROLES. :x:');
+  if (message.member.roles.cache.find(r => r.id === "898041754564046869") == null) return message.reply("sorry, but it looks like you're too much of a boomer to run this command.");
+  if (args.length < 1) {
+      message.channel.send('', {
+          embed: new Discord.MessageEmbed().setColor(0x00A2E8)
+              .setDescription(`Correct usage ${config.DiscordBot.Prefix}votemute <@user|userID> [Time : 5m] [Reason : unspecified]`).setFooter('<required> [optional]')
+      })
+      return;
+  }
+  let target = message.guild.members.cache.get(args[0].match(/[0-9]{18}/).length == 0 ? args[0] : args[0].match(/[0-9]{18}/)[0])
+  let reason = args.slice(2).join(' ') || `unspecified`;
+  let time = ms(args[1]) || 300000;
+
+  if (target == null) return message.reply("please specify a valid user.");
+  if (time > 7200000) time = 7200000; // If the time value is over 2 hours
+  if (time < 5000) time = 5000; // If the time value is less than 5 seconds
+
+  const yes = '👍';
+  const filter = (reaction, user) => reaction.emoji.name === yes;
```

Comment:
you could just run the command inside the spam channel then use bots to react the message.

Reply:
how is this resolved?

Reply:
command works only in lounge/dono-lounge (idk id of beta-lounge)

Reply:
no?, the command works everywhere